src/Mvc/Mvc.ViewFeatures/src/DefaultHtmlGeneratorExtensions.cs

```diff
@@ -1,12 +1,27 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures
 {
+    /// <summary>
+    /// Static class that adds extension methods to <see cref="IHtmlGenerator"/>.
```

Comment:
```suggestion
    /// Static class that adds extension methods to <see cref="IHtmlGenerator"/>. This class cannot be inherited.
```

Same as above with this and other static classes.